easybuild/easyconfigs/b/Bowtie/Bowtie-1.3.1-GCC-11.2.0.eb

```diff
@@ -0,0 +1,30 @@
+##
+# This is a contribution from DeepThought HPC Service, Flinders University, Adelaide, Australia
+# Homepage: 	https://staff.flinders.edu.au/research/deep-thought
+#
+# Authors::	Robert Qiao <rob.qiao@flinders.edu.au>
+# License::	Artistic v2.0
+#
+# Notes::
+##
+
+name = 'Bowtie'
+version = '1.3.1'
+
+homepage = 'http://bowtie-bio.sourceforge.net/index.shtml'
+description = """Bowtie is an ultrafast, memory-efficient short read aligner.
+ It aligns short DNA sequences (reads) to the human genome."""
+
+toolchain = {'name': 'GCC', 'version': '11.2.0'}
+toolchainopts = {'pic': True, 'cstd': 'gnu++98'}
+
+source_urls = ['https://sourceforge.net/projects/bowtie-bio/files/bowtie/%(version)s/']
+sources = ['%(namelower)s-%(version)s-src.zip']
+checksums = ['e23517aa53846ef828172be911750cd05748522117efcbbe5a36f3241fb40761']
+
+dependencies = [
+    ('tbb', '2021.4.0'),
```

Comment:
note: this was downgraded to 2020.3 in #14800, to ensure this can be used as an (indirect) dependency in easyconfigs using a `2021b` toolchain